site/en/docs/devtools/console/api/index.md

```diff
@@ -73,6 +73,24 @@ console.countReset();
 console.countReset('coffee');
 ```
 
+## console.createTask(name) {: #createtask }
+
+{% Aside 'gotchas' %}
+This method implements the Async Stack Tagging API. If you're using a framework that handles the scheduling and async execution, you don't need to use this method. In that case, it is up to the framework to implement the API.
```

Comment:
I'd word this as "This method is known as the Async Stack Tagging API. It you're using a framework or abstraction that already started using this under the hood to keep track of scheduling and async execution, then you don't need to use this method directly."

Reply:
I'd remove "In that case, it is up to the framework to implement the API.". Frameworks don't _implement_ this API, they use it. We (DevTools) implemented the API.